services/scan/src/store.ts

```diff
@@ -859,18 +821,11 @@ export class Store {
   }
 
   /**
-   * Exports all CVR JSON data to a stream.
+   * Yields all sheets in the database that would be included in a CVR export.
    */
-  async exportCvrs(
-    writeStream: Writable,
-    options: { skipImages?: boolean; orderBySheetId?: boolean } = {}
-  ): Promise<void> {
-    const electionDefinition = this.getElectionDefinition();
-
-    if (!electionDefinition) {
-      throw new Error('no election configured');
-    }
-
+  *forEachResultSheet(options?: {
+    orderBySheetId?: boolean;
+  }): Generator<ResultSheet> {
```

Comment:
Do we need testing for this, or do you consider it tested via the `export.test.ts` testing?

Reply:
I think it's reasonable to have at least some basic testing in `store.test.ts`. I'll add that.